src/Unused/GitContext.hs

```diff
@@ -29,5 +29,9 @@ logToGitContext =
 
 gitLogSearchFor :: Int -> [String] -> IO GitOutput
 gitLogSearchFor commitCount ts = do
-  (_, results, _) <- P.readProcessWithExitCode "git" ["log", "-G", L.intercalate "|" ts, "--oneline", "-n", show commitCount] ""
-  return $ GitOutput results
+    (_, results, _) <- P.readProcessWithExitCode "git" (gitCommand commitCount ts) ""
+    return $ GitOutput results
+  where
+    gitCommand :: Int -> [String] -> [String]
+    gitCommand commitCount [t] = ["log", "-S", t, "--oneline", "-n", show commitCount]
+    gitCommand commitCount ts = ["log", "-G", L.intercalate "|" ts, "--oneline", "-n", show commitCount]
```

Comment:
These bindings for `commitCount` and `ts` shadow the same bindings in `gitLogSearchFor commitCount ts` above.

Do you want me to rename these lines, or just leave them?

Reply:
@gabebw ah, nice catch - mind making them `commitCount'` and `ts'`, respectively?

Reply:
@joshuaclayton Done!